docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/tests/conftest.py

```diff
@@ -0,0 +1 @@
+# Define test fixtures here.
```

Comment:
What do you think about adding a default fixture to load in initial content here, like you did over in StormStore?

Reply:
That's a great idea.